data/linus.txt

```diff
@@ -0,0 +1 @@
+[{"type":"linus.task.Deadline","by":"2023-08-31","description":"task 2","isDone":false},{"type":"linus.task.Deadline","by":"2022-01-01","description":"TEST","isDone":true},{"type":"linus.task.ToDo","description":"ggg","isDone":true},{"type":"linus.task.ToDo","description":"johjh","isDone":true},{"type":"linus.task.Deadline","by":"2100-03-02","description":"TEST","isDone":true},{"type":"linus.task.ToDo","description":"TEST","isDone":true},{"type":"linus.task.ToDo","description":"adfasf adsfas","isDone":false}]
```

Comment:
This might be beyond the scope of the review, but perhaps the data folder should not be tracked in git?